### PR TITLE
VAT item added

### PR DESCRIPTION
**Pull request by sona-podholova**
# TL;DR

Not only is it possible to save VAT, but also some other minor bugs are fixed

# Proof
image

# Merge request checklist

Please check if your merge request fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (yarn build) was run locally and any changes were pushed
- [x] API (yarn api:dev) runs locally and any fixes were made for failures
- [x] Admin Client (cd clients/admin && yarn start) runs locally and any fixes were made for failures

# Merge request type

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

# Before this merge request
Some items either did not appear or were not sent

# How you fixed what was wrong
Everything is displayed and sent

### CODE REVIEW

**Review by davidpodhola — APPROVED** [review]
Cool, thank you, let's work on the tests too.
